src/components/patch/PatchContainer.tsx

```diff
@@ -49,6 +56,24 @@ export class PatchContainer extends React.Component<Props, State> {
 
     return (
       <div>
+        <div className="banner-container" hidden={this.state.bannerIsHidden}>
```

Comment:
I'd like to see this in a separate component so that it can be reused on future converted pages. The message should be configurable, and whether or not it's been acknowledged should be tracked separately. There should also be tests of it